### PR TITLE
Add config for multiple controllers

### PR DESCRIPTION
**Pull request by Flova**
## Proposed changes
Cleanup button codes.

### CODE REVIEW

**Review by SammyRamone — COMMENTED** [review]
It looks like you changed the head control system from an incremental one to a direct mapping of the control input. Is this nicely controllable? I think its maybe hard to keep the head looking at a direction when you are also walking

File: bitbots_teleop/src/bitbots_teleop/joy_node.py [hunk 1]
```diff
@@ -22,12 +21,63 @@ def __init__(self):
         log_level = rospy.DEBUG if rospy.get_param("/debug_active", False) else rospy.INFO
         rospy.init_node("joy_to_twist", log_level=log_level, anonymous=False)
 
-        self.head_pan_pos = 0
-        self.head_tilt_pos = 0
+        controller_configs = {
```
Comment (16stelter) [inline bitbots_teleop/src/bitbots_teleop/joy_node.py:24]: Please move the configs into a seperate file

**Review by 16stelter — DISMISSED** [review]
Assuming, you've tested this, especially the parameters, this looks fine. It would be more readable though, if you moved the controller configuration into a config file.

**Comment by Flova** [comment]
> It looks like you changed the head control system from an incremental one to a direct mapping of the control input. Is this nicely controllable? I think its maybe hard to keep the head looking at a direction when you are also walking

I tried both and with a proper controller is was nicer to control it absolutely. We have a quite large fov and dont need to aim very precisely. Imo this is a good trade off when we keep in mine that the absolute control allows us to look left and right very quickly to orient our selfs or some thing like that.

**Review by jgueldenstein — APPROVED** [review]
tested with xbox controller in simulation

**Pull request merged**